### DISCUSSION TITLE
on AlphaIDE使用体验

### DISCUSSION THREAD

**Discussion by pengbo** [meta-soul/MetaSpore#138] [General]
1. 产品能够很好的集成了云的能力，覆盖了数据分析、机器学习等场景的核心流程，基本做到了一站式

2. 但有一些产品细节需要继续打磨
![image](https://user-images.githubusercontent.com/1247590/172608710-cc6a0507-17ee-4043-9ce4-ce19b49bd95b.png)

![image](https://user-images.githubusercontent.com/1247590/172608744-42783746-ff00-4341-8f46-84132b18307b.png)

此外，页面布局感觉还是有一些比例问题，比如 header、侧边以及需要重点突出的内容模块大小，还是可以再优化一下

3. 技术栈看起来是 spark 为主，主流的 hadoop、flink、tensorflow 会考虑集成么

**Comment by intelligencegear** [comment]
感谢提供的系统交互优化建议，MetaSpore目前是PyTorch on Spark，LakeSoul除了Spark之外，未来也会支持Flink。
